tests/integration_tests/test_sklearn.py

```diff
@@ -9,6 +9,18 @@
 import numpy as np
 
 
+def test__is_arraylike():
```

Comment:
Please add a type hint and a blank line (c.f., https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#empty-lines-after-type-hints).

And also, we did add just `test` to the prefix of test functions if the target functions start from `_`.

```suggestion
def test_is_arraylike() -> None:

```